src/main/java/seedu/address/commons/util/DateUtil.java

```diff
@@ -40,4 +40,17 @@ public static String format(Date date) {
         return DATE_FORMATTER.format(date);
     }
 
+    /**
+     * Adds a number of days to a Date object.
+     *
+     * @param date The Date object to add days to.
+     * @param days The number of days to add.
+     * @return The new Date object.
+     */
+    public static Date addDay(Date date, int days) {
```

Comment:
This is not urgent, but I think we can refactor date representation in our project to use `LocalDate` or `LocalDateTime`. It is a more recent API that has more inbuilt methods such as `plusDays()` which can be used to avoid weird methods such as this,